src/Caliburn.Micro.Core.Tests/Caliburn.Micro.Core.Tests.csproj

```diff
@@ -1,5 +1,5 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
-
+﻿<Project>
+  <Sdk Name="MSBuild.Sdk.Extras" Version="2.0.24" />
```

Comment:
Can we drop the version from all the projects and bump the number in [global.json](https://github.com/Caliburn-Micro/Caliburn.Micro/blob/master/src/global.json) to ensure it's consistent across all projects?

Reply:
Probably, I honestly didn't even know it was there I'll fix that up switch the reference back to the 4.6 folder (actually how I originally did it but seemed like convention.